recipes/merge-args/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "merge-args" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/merge_args-{{ version }}.tar.gz
+  sha256: 79b01449801757f6ef2a24520b90fa270bc1a5296adf731a899afa950e6f1545
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >3.6
```

Comment:
Upstream also includes `3.5` in case you want to include it: https://github.com/Kwpolska/merge_args/blob/155778f4013138eaa5b5e4cf1a2c539525326fc3/.github/workflows/ci.yml#L10